buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidLibTarget.groovy

```diff
@@ -2,13 +2,15 @@ package com.uber.okbuck.core.model.android
 
 import com.android.build.gradle.api.BaseVariant
 import com.android.manifmerger.ManifestMerger2
+import com.uber.okbuck.core.util.KotlinUtil
 import org.gradle.api.Project
-
 /**
  * An Android library target
  */
 class AndroidLibTarget extends AndroidTarget {
 
+    private static final String KOTLIN_EXTENSIONS_OPTION = "plugin:org.jetbrains.kotlin.android:"
```

Comment:
I don't see the constant in the kotlin gradle plugin but I do see it in kotlin android extensions itself. But this would require us to add that as a dependency, would you want to do that just for the constant?